sigs.yml

```diff
@@ -0,0 +1,502 @@
+      value: 1tCyoQK49WVcE-x8oryZOTTToFm7sIeUhxFPm9g-qL1k
+    chat:
+      - type: slack
+        name: '#otel-js'
+        id: C01NL1GRPQR
+      - type: other
+        name: GitHub Discussions
+        link: https://github.com/open-telemetry/opentelemetry-js/discussions
+    invites: calendar-js
+    tcSponsors: []
+    gcLiaison:
+    - name: 'Daniel Dyla'
+      github: dyladan
+  - name: 'PHP: SDK'
+    meeting: Every Wednesday at 10:30 PT
+    notes:
+      type: gDoc
+      value: 1WLDZGLY24rk5fRudjdQAcx_u81ZQWCF3zxiNT-sz7DI
+    chat:
+      - type: slack
+        name: '#otel-php'
+        id: C01NFPCV44V
+    invites: calendar-php
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Python: SDK'
+    meeting: Every Thursday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs
+    chat:
+      - type: slack
+        name: '#otel-python'
+        id: C01PD4HUVBL
+    invites: calendar-python
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Ruby: SDK'
+    meeting: Every Tuesday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 1D15bO8o340sQm2CVZiukEJuCO_XMMHKPuTznoEhyFqE
+    chat:
+      - type: slack
+        name: '#otel-ruby'
+        id: C01NWKKMKMY
+      - type: other
+        name: GitHub Discussions
+        link: https://github.com/open-telemetry/opentelemetry-ruby/discussions
+    invites: calendar-ruby
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Rust: SDK'
+    meeting: Every Tuesday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4
+    chat:
+      - type: slack
+        name: '#otel-rust'
+        id: C03GDP0H023
+      - type: other
+        name: Gitter
+        link: https://gitter.im/open-telemetry/opentelemetry-rust
+    invites: calendar-rust
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Swift: SDK'
+    meeting: Every Thursday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 1LugL8r4bAkbTxZ1Gq_6j_8SDv1LuZaNeUdR-QLN4d48
+    chat:
+      - type: slack
+        name: '#otel-swift'
+        id: C01NCHR19SB
+    invites: calendar-swift
+    tcSponsors: []
+    gcLiaison: []
+  - name: Communications (Website, [Documentation](https://opentelemetry.io/docs/),
+      etc.)
+    meeting: Every other week on Monday at 10:00 PT
+    notes:
+      type: gDoc
+      value: 1wW0jLldwXN8Nptq2xmgETGbGn9eWP8fitvD5njM-xZY
+    chat:
+      - type: slack
+        name: '#otel-comms'
+        id: C02UN96HZH6
+    invites: calendar-comms
+    tcSponsors: []
+    gcLiaison:
+    - name: 'Severin Neumann'
+      github: svrnm
+  - name: End-User WG
+    meeting: Every other week on Thursday at 10:00 PT
+    notes:
+      type: gDoc
+      value: 1e-UNZA3Tuno9b53RQbe--whUcO0VIXF3P81oXsrBK6g
+    chat:
+      - type: slack
+        name: '#otel-user-research'
+        id: C01RT3MSWGZ
+    invites: calendar-user-research
+    tcSponsors: []
+    gcLiaison:
+    - name: Daniel Gomez Blanco
+      github: danielgblanco
```

Comment:
@danielgblanco please confirm that you are going to be the liaison for this SIG

Reply:
I can confirm this 👍